src/contexts/settings/hooks.tsx

```diff
@@ -50,7 +50,7 @@ export const useTheme = (initialState:ThemeState) => {
     } else if (themeDictionary[themeSelection]) {
       setTheme(themeSelection as Theme);
     } else {
-      setTheme('light');
+      // setTheme('light');
```

Comment:
Instead of commenting out we can handle edge cases in case something doesn't go as plan with:
```
setTheme('dark')
```

Reply:
Changed and pushed again!